src/Servers/IIS/IIS/test/Common.FunctionalTests/ShutdownTests.cs

```diff
@@ -211,7 +253,6 @@ public async Task AppOfflineDropped_CanRemoveAppOfflineAfterAddingAndSiteWorks(H
         [ConditionalTheory]
         [InlineData(HostingModel.InProcess)]
         [InlineData(HostingModel.OutOfProcess)]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/7075")]
```

Comment:
Missed unquaranting?

Reply:
Yeah, and I did do work on the test in the pass.